modules/ROOT/pages/_partials/sync/creating-a-voyager-server-dockerfile-for-use-with-openshift.adoc

```diff
@@ -0,0 +1,59 @@
+[id="creating-a-voyager-server-dockerfile-for-use-with-openshift-{context}"]
+= Creating a voyager server container image for use with OpenShift
+
+This section describes how to create a container image using a local voyager server repo.
+
+Although it is not covered in this section, you can add the Apollo Voyager Client.
+For more information, see xref:server-getting-started-with-an-example-data-source-{context}[Getting started with an example data source].
+
+.Prerequisites
+
+* You have cloned the link:https://github.com/aerogear/voyager-server[Apollo Voyager Server] locally.
+* Docker is installed and is running on your local machine. Use `sudo systemctl start docker` to start docker. As an alternative, use `podman`, or `buildah`.
```

Comment:
That `sudo systemctl start docker` -- what OS are we assuming here?

- MacOS & Windows: don't use systemd, so it doesn't apply
- Fedora: now defaults to `podman` instead of docker on all supported versions (I think)
- RHEL: RHEL8 defaults to `podman`, doesn't even ship docker I think. `podman` should be available and work fine on RHEL7 (I haven't checked).
- Debian/Ubuntu/Arch then? :D